build.rs

```diff
@@ -1,3 +1,15 @@
+use std::{env, process::Command};
+
+#[allow(clippy::disallowed_macros)] // for println!
 fn main() {
     lalrpop::process_root().unwrap();
+    let output = Command::new("git")
```

Comment:
Can we make this optional info? (along with the `CARGO_PKG_VERSION` query).

That way we can still run / build this on machines that don't have git, or similar.